source/wp-content/themes/wporg-news-2021/sass/post/_header.scss

```diff
@@ -1,4 +1,240 @@
-.header-archive {
+.site-header {
```

Comment:
Could this be modularized into 2-4 files? It seems like a lot for 1 file, and might have different/unrelated chunks that could be separated.

Is `post/header/scss` the right file, it seems like this applies to pages/archives/etc? i'd assume this file would just have `entry-header` stuff

Reply:
🤔  it seems like we might want both `posts/` and `components/`? Most of the stuff in `post/` right now seems legitimately tied to posts (and only posts).

Stuff like this isn't directly tied to posts, though